cdap-ui/app/styles/fonts/icomoon.svg

```diff
@@ -35,4 +35,5 @@
 <glyph unicode="&#xe61b;" glyph-name="ETLRealtime" d="M1024 362.13l-87.186 189.879-121.125-170.569 46.738-4.315c-27.648-135.022-130.999-244.297-266.24-275.895-141.166-32.841-286.647 22.162-370.834 140.507-16.823 23.479-49.445 29.33-73.070 12.361-23.625-16.75-29.111-49.518-12.288-72.997 87.991-123.758 227.694-194.048 374.491-194.048 35.035 0 70.363 4.023 105.472 12.215 178.907 41.765 314.953 188.416 346.99 368.128l57.051-5.266zM161.499 518.875c27.502 134.875 130.926 244.151 266.24 275.822 141.531 33.207 286.72-22.089 370.907-140.361 16.677-23.552 49.225-29.257 72.997-12.361 23.552 16.75 29.111 49.445 12.361 72.997-108.837 153.161-297.179 224.622-480.037 181.76-179.054-41.911-314.953-188.343-347.063-368.128l-56.905 5.266 87.186-190.025 121.125 170.642-46.811 4.389zM515.072 665.307c-16.384 0-29.696-13.312-29.696-29.696v-100.206c-36.791-12.581-63.781-46.373-63.781-87.406 0-51.639 41.838-93.55 93.477-93.55 41.106 0 74.898 26.99 87.479 63.854h153.381c16.384 0 29.696 13.312 29.696 29.696s-13.385 29.696-29.696 29.696h-153.381c-9.362 27.355-30.427 48.347-57.783 57.71v100.206c0 16.384-13.312 29.696-29.696 29.696z" />
+<glyph unicode="&#xe61c;" glyph-name="SchemaEdge" d="M100.96 755.68v-718.688h718.72v-100.928h-718.72c-27.264 0-50.88 9.984-70.912 30.016s-30.048 43.648-30.048 70.88v718.72h100.96zM993.952 929.92c-20.032 20.032-43.648 30.048-70.912 30.048h-583.328c-27.264 0-51.296-10.016-72.096-30.048s-31.232-43.68-31.232-70.912v-615.328c0-27.296 10.432-51.296 31.232-72.096 20.832-20.832 44.864-31.264 72.096-31.264h583.328c27.264 0 50.88 10.4 70.912 31.264 20.032 20.8 30.048 44.8 30.048 72.096v615.328c0 27.232-10.016 50.88-30.048 70.912zM565.248 460.256v68.96h137.984v-68.96h-137.984zM703.264 392v-68.96h-137.984v68.96h137.984zM565.248 600v68.96h137.984v-68.96h-137.984zM565.248 737.216v68.96h137.984v-68.96h-137.984zM358.176 806.176h138.112v-68.96h-138.112v68.96zM358.176 668.96h138.112v-68.96h-138.112v68.96zM358.176 529.216h138.112v-68.96h-138.112v68.96zM358.176 392h138.112v-68.96h-138.112v68.96zM902.144 323.040h-129.92v68.96h129.92v-68.96zM902.144 460.256h-129.92v68.96h129.92v-68.96zM902.144 600h-129.92v68.96h129.92v-68.96zM902.144 737.216h-129.92v68.96h129.92v-68.96z" />
 </font></defs></svg>
```

Comment:
@anwar6953 This is a generated file that will most likely not be opened by anyone. Not sure if it is required here